src/main/resources/org/jfrog/hudson/pipeline/declarative/steps/InteractivePromotionStep/help-displayName.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    If set, the promotion window will display this label instead of the build name and number.
```

Comment:
"the promotion window displays this label..." is better style -- present tense rather than future.

Reply:
@StackScribe
Thank you 👍, I will correct that.